### DISCUSSION TITLE
Confirmation to delete a cell with 'x'?

### DISCUSSION THREAD

**Discussion by diego-pm** [microsoft/vscode-jupyter#9922] [New Ideas]
It sometimes happens to me that I press the letter 'x' and accidentally remove a cell without noticing. Could this be solved by adding a confirmation dialogue to delete the cell?

**Comment by rchiodo** [comment]
Sorry but 'x' shouldn't remove anything? What do your keybindings say? 

It should take 'd d' to remove a cell.

Perhaps you were holding down 'CTRL' at the time?

**Reply by diego-pm** [reply]
'x' cuts the cell, but if you forget it or don't see it, is like removing the cell.

**Reply by rchiodo** [reply]
Ah it's the same a cut. That seems weird. 

Well you can change the keybinding yourself it you don't want it. 

This is what I did:

![image](https://user-images.githubusercontent.com/19672699/168120578-295d7785-d428-47b9-9301-70a9090dc97e.png)

Now I have to hit ctrl+x to cut the cell.
